cmake/modules/FindSSE.cmake

```diff
@@ -40,6 +40,26 @@ if(CMAKE_SYSTEM_NAME MATCHES "Linux")
      string(COMPARE EQUAL "avx2" "${_SSE_THERE}" _AVX2_TRUE)
      CHECK_CXX_ACCEPTS_FLAG("-mavx2" _AVX2_OK)
    endif()
+elseif(CMAKE_SYSTEM_NAME MATCHES "Android")
+  if(CPU MATCHES "x86_64" OR CPU MATCHES "i.86")
+    set(_SSE_TRUE TRUE)
+    set(_SSE2_TRUE TRUE)
+    set(_SSE3_TRUE TRUE)
+    set(_SSSE3_TRUE TRUE)
+    set(_SSE41_TRUE TRUE)
+    set(_SSE42_TRUE TRUE)
+    set(_AVX_TRUE TRUE)
+    set(_AVX2_TRUE TRUE)
```

Comment:
are you sure all CPUs in x86 based android devices support avx(2)?
I'd be much more conservative here and only enable what kodi requires to build by default. <= SSE2 should be enough.